app/controllers/admin/payments_controller.rb

```diff
@@ -133,26 +133,37 @@ def create_invoice(csv, payments, payment_type, start_date, end_date = nil)
         p = Participant.where(member: payment.member, activity_id: activity_id).first
         # now create every transaction row with the following date: Blank, ledger account,
         # transaction description, VAT number, amount, cost_location ()
-        csv <<  if p.activity.group.nil? || Group.first.ledgernr.blank?
-                  ["", "1302", "#{ p.activity.name } - #{ p.member_id }", p.activity.VAT,
-                   p.currency, ""]
-                else
-                  ["", p.activity.group.ledgernr, "#{ p.activity.name } - #{ p.member_id }",
-                   p.activity.VAT, p.currency, p.activity.group.cost_location]
-                end
+        csv << if Activity.where(id: activity_id).description == "Lidmaatschap"
```

Comment:
```suggestion
        csv << if activity_id == Settings['intro.membership']
```

Came across this in `lib/tasks/admin.rake`, seems like the way to go.

Reply:
Thought about this aswell only issue being. In the hypothetical case you would create an invoice for introduction of previous years it would fail on this condition

Reply:
I don't know if that is a big issue;
If the old membership activity does not have a group, it will become a 1302, which is not wrong, just less informative.
If the old membership activity does have a group, it should be set correctly anyhow.